app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/tei_data/TEIDataPresenterImpl.java

```diff
@@ -274,18 +260,7 @@ public void seeDetails(View sharedView, DashboardProgramModel dashboardProgramMo
 
     @Override
     public void onScheduleSelected(String uid, View sharedView) {
-        String title = String.format("%s %s - %s",
-                dashboardModel.getTrackedEntityAttributeValueBySortOrder(1) != null ? dashboardModel.getTrackedEntityAttributeValueBySortOrder(1) : "",
-                dashboardModel.getTrackedEntityAttributeValueBySortOrder(2) != null ? dashboardModel.getTrackedEntityAttributeValueBySortOrder(2) : "",
-                dashboardModel.getCurrentProgram() != null ? dashboardModel.getCurrentProgram().displayName() : view.getContext().getString(R.string.dashboard_overview)
-        );
         Intent intent = ScheduledEventActivity.Companion.getIntent(view.getContext(),uid);
-        /*Intent intent = new Intent(view.getContext(), EventDetailActivity.class);
-        Bundle extras = new Bundle();
-        extras.putString("EVENT_UID", uid);
-        extras.putString("TOOLBAR_TITLE", title);
-        extras.putString("TEI_UID", teiUid);
-        intent.putExtras(extras);*/
         ActivityOptionsCompat options = ActivityOptionsCompat.makeSceneTransitionAnimation(view.getAbstractActivity(), sharedView, "shared_view");
         view.openEventDetails(intent, options.toBundle());
     }
```

Comment:
Same as previous comment

Reply:
Agreed, but not for now.

Reply:
![ok](https://media.giphy.com/media/3o7abKhOpu0NwenH3O/giphy.gif)